vendor/github.com/projectatomic/buildah/vendor.conf

```diff
@@ -4,8 +4,8 @@ github.com/BurntSushi/toml master
 github.com/containerd/continuity master
 github.com/containernetworking/cni v0.7.0-alpha1
 github.com/seccomp/containers-golang master
-github.com/containers/image 216acb1bcd2c1abef736ee322e17147ee2b7d76c
-github.com/containers/storage 17c7d1fee5603ccf6dd97edc14162fc1510e7e23
+github.com/containers/image 5df44e095ed826fbe2beeaabb329c749d7d6c3b6
+github.com/containers/storage 9fcbb57eb6c732e7b67003bb8ed861f169d33d63
```

Comment:
Wasn't expecting to see this file in this PR and the container/storage id's differ...

Reply:
Yes not sure why it vendored.  I don't think we have updated the vendor of container/storage in buildah to the latest yet.